code/__HELPERS/unsorted.dm

```diff
@@ -1260,8 +1260,8 @@ var/list/WALLITEMS = list(
 	"/obj/machinery/newscaster", "/obj/machinery/firealarm", "/obj/structure/noticeboard", "/obj/machinery/door_control",
 	"/obj/machinery/computer/security/telescreen", "/obj/machinery/embedded_controller/radio/simple_vent_controller",
 	"/obj/item/weapon/storage/secure/safe", "/obj/machinery/door_timer", "/obj/machinery/flasher", "/obj/machinery/keycard_auth",
-	"/obj/structure/mirror", "/obj/structure/closet/fireaxecabinet", "/obj/machinery/computer/security/telescreen/entertainment",
-	"obj/structure/sign"
+	"/obj/structure/mirror", "/obj/structure/closet/fireaxecabinet", "/obj/structure/closet/fireaxecabinet/empty", 

```

Comment:
It didn't work when I only had the parent, though I changed something else that also may have been the cause. Also, telescreen and telescreen/entertainment are both in there. Should I take that out too?